index.html

```diff
@@ -85,8 +87,8 @@ <h2>OUR SERVICES</h2>
         <h3 class="trending-now">TRENDING NOW</h3>
         <div class="line"></div>
         <div class="products-container"></div>
```

Comment:
- [ ] (OPTIONAL) Great job generating this section dynamically from JavaScript 👍 It would be great if you could add the chess-like image decoration to each image so as to follow the design requirements 💯 
- Your design:
![Annotation 2022-05-18 105830](https://user-images.githubusercontent.com/85113567/169015364-59d642ab-e3a5-405e-81f0-fc62e636729e.jpg)
- Required design:
![Annotation 2022-05-18 105850](https://user-images.githubusercontent.com/85113567/169015417-a5e77db5-bfaf-45ed-aacb-e4daf0d29684.jpg)